paper-slider.html

```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```

Comment:
Yeah it's odd that it'st async. Presumably there was originally a reason for that...

I'll test it out next week when I'm physically back at my machine in SFO.